docs/git/homework.md

```diff
@@ -78,13 +78,12 @@ Again, go to this repository and go to the "Pull Requests" tab.
 
 https://github.com/CodeYourFuture/GitHomeworkFixErrors
 
-Here you will find all of your Classmates answers to the homework. 
+Here you will find all of your Classmates answers to the homework.
 
-1. Choose two pull requests
-2. Make sure they don't have any comment on them already from other students
-3. Leave comments on any of the questions you think they have got wrong. Not sure if their answer is right? Ask them to check it and confirm.
-4. Leave a comment saying you have finished you're review
+1. Choose two pull requests with no, or few, pull requests
```

Comment:
```suggestion
1. Choose two pull requests with no, or few, comments.
```

Reply:
Lol, that was me adding a new mistake while fixing another one 🤦‍♂️